### ISSUE TITLE
Change first state's feature / symbol

### ISSUE THREAD

**Issue by tbazin** [wangsix/vmo#11]

**Comment by wangsix** [comment]
I think it was setup like that so when casting something like y = numpy.array(oracle.symbol) you don`t have to worry about the type, but at the end I think it`s an implementation choice since most of the time we start from oracle.symbol[1:]. What do you think?

**Comment by tbazin** [comment]
Yes, I can see that. But changing it to `None` actually **doesn't** pose a problem when doing such a cast (since Python's lists and arrays don't have a single-type content requirement).

And in my opinion, it is better to worry about the types (unless You're working with inheritance or subtyping and it actually makes sense not to want to worry about the types, since it is all properly handled by the language) when something is so implementation dependant and not really *semantic*.
(Plus, in that perspective, one would constrain all symbols to be of type `int`, which isn't the only meaningful way to see things. Why not `char`s for instance?)

**Comment by tbazin** [comment]
Alright, it does actually mess up the codebase, You were right.
And then again, the typing system in Python being what it is, Your way of seeing things also makes sense.

Might have to think more about that one before changing anything!
